crates/forge/tests/cli/soldeer.rs

```diff
@@ -98,13 +101,7 @@ forgesoldeer!(install_dependency_git_commit, |prj, cmd| {
 
     let foundry_file = prj.root().join("foundry.toml");
 
-    cmd.arg("soldeer")
-        .args([command, dependency, git, rev_flag, commit])
-        .assert_success()
-        .stdout_eq(str![[r#"
-🦌 Running [..]oldeer install 🦌
-...
-"#]]);
+    cmd.arg("soldeer").args([command, dependency, git, rev_flag, commit]).assert_success();
```

Comment:
please add `stdout_eq(str![[""]])` and then run the tests with `SNAPSHOTS=overwrite` to auto generate them

Reply:
oh, did not know that, thanks, i'm gonna do that once i figure out why the tests are failing on windows in the Actions...

Reply:
done